examples/cookies/README.md

```diff
@@ -0,0 +1,24 @@
+# Cookie example
+
+We can achieve cookies by different ways:
+
+1. Using 'key' parameter. Fasthttp will search using key in cookie field.
+
+2. You can parse request header 'Set-Cookie' field.
+
+3. You can use 'PeekCookie' from ResponseHeader using key. This is the long way to get cookie instead of using 'Cookie' structure.
```

Comment:
`headers.PeekCookie` and `headers.Cookie` do exactly the same thing except `PeekCookie` doesn't parse the headers if they aren't parsed yet.